cypress.config.js

```diff
@@ -0,0 +1,53 @@
+const { defineConfig } = require("cypress");
+
```

Comment:
@Rampraesath What is this for?

Reply:
as per discussion i have removed this file